### DISCUSSION TITLE
Can't register or login - indefinite wait

### DISCUSSION THREAD

**Discussion by dharvey242** [sysadminsmedia/homebox#336] [Support]
### First Check

- [x] This is not a feature request
- [x] I added a very descriptive title to this issue.
- [x] I used the GitHub search to find a similar issue and didn't find it.
- [x] I searched the documentation, with the integrated search.
- [x] I already read the docs and didn't find an answer.
- [x] I can replicate the issue inside the Demo install.

### Homebox Version

Version: v0.10.3 ~ Build: e8449b3a7363a6cfd5bc6151609e6d2d94b4f7d8

### What is the issue you are experiencing?

Sorry Noob question.  I've followed the installation process to the letter - however on my RPI when I start docker-compose - everything appears to work but when i try and register a new user after clicking submit it just hangs.   It never seems to complete process.  I've installed the .exe on my windows machine so I know what to expect.  

However my RPI never seems to get past the initial register stage.  I can only assume its potentially unable to store the credentials locally?

All help is appreciated.

I get these errors in the log

`3:06PM ERR ../go/src/app/app/api/handlers/v1/v1_ctrl_user.go:39 > failed to register user error="ent: constraint failed: constraint failed: UNIQUE constraint failed: users.email (2067)"
3:06PM ERR ../go/src/app/internal/web/mid/errors.go:31 > ERROR occurred error="ent: constraint failed: constraint failed: UNIQUE constraint failed: users.email (2067)" req_id=0d085093d08f/U3Cq9psLR9-000091
3:06PM INF ../go/src/app/internal/web/mid/logger.go:41 > request finished method=POST path=/api/v1/users/register rid=0d085093d08f/U3Cq9psLR9-000091 status=500
3:06PM INF ../go/src/app/internal/web/mid/logger.go:36 > request received method=POST path=/api/v1/users/register rid=0d085093d08f/U3Cq9psLR9-000092
`

### How can the maintainer reproduce the issue?

Install on RPI

### Deployment

Docker (Linux)

### OS Architechture

ARM64

**Comment by tankerkiller125** [comment]
That log entry seems to imply that the user already exists, and that's why it's unable to save

**Reply by dharvey242** [reply]
Thanks for the reply.  I've tried to login directly using the same details and I get the same result. Once I click login it hangs. Also tried different registration information and again same experience. It's as if it can't register/authenticate the information with the server.

**Reply by tankerkiller125** [reply]
I've just now noticed the version your running is significantly out of date, this fork is currently on v0.15.2

I'd highly recommend updating to the latest version and trying again.

**Reply by dharvey242** [reply]
Thanks.  BCM2837B0 Arm A53.  Buster version of Debian.   Also tried it on the same processor but latest rpi image and the same thing.

**Comment by dharvey242** [comment]
I had this line in my docker compose file

image: ghcr.io/sysadminsmedia/homebox:latest

Would that not pull the latest version? 

I'm pretty sure I've tried both the previous/original config and this latest version and the server response is the same.  I'll try it again using the line above and report back. Though I'm sure I use or have used it previously.

**Comment by dharvey242** [comment]
I've upgraded to the latest versions in this GIT - I even updated the RPI OS to the latest verson and i'm still getting the same errors no matter what email, username combination i use.  The timeout 500 error seems to suggest it can't even get to the server to validate anthing.
All help is much appreciated.

homebox    | 8:17AM ERR ../go/src/app/app/api/handlers/v1/v1_ctrl_user.go:39 > failed to register user error="ent: constraint failed: constraint failed: UNIQUE constraint failed: users.email (2067)"
homebox    | 8:17AM ERR ../go/src/app/internal/web/mid/errors.go:31 > ERROR occurred error="ent: constraint failed: constraint failed: UNIQUE constraint failed: users.email (2067)" req_id=62e53ffbdf44/CbUjY697WU-000059homebox    | 8:17AM INF ../go/src/app/internal/web/mid/logger.go:41 > request finished method=POST path=/api/v1/users/register rid=62e53ffbdf44/CbUjY697WU-000059 status=500

**Reply by sgiraz** [reply]
I'm experiencing the same issue. Everything looks good on Windows, but I cannot say the same on my RPI 3B+.
I'm running `v0.15.2`


```
homebox  | 3:55PM INF ../go/src/app/internal/web/mid/logger.go:36 > request received method=POST path=/api/v1/users/register rid=2b2443a73146/Ka5V5SeNcl-000165
homebox  | 3:55PM INF ../go/src/app/internal/web/mid/logger.go:36 > request received method=GET path=/api/v1/status rid=2b2443a73146/Ka5V5SeNcl-000166
homebox  | 3:55PM INF ../go/src/app/internal/web/mid/logger.go:41 > request finished method=GET path=/api/v1/status rid=2b2443a73146/Ka5V5SeNcl-000166 status=200
homebox  | 3:55PM ERR ../go/src/app/app/api/handlers/v1/v1_ctrl_user.go:39 > failed to register user error="ent: constraint failed: constraint failed: UNIQUE constraint failed: users.email (2067)"
homebox  | 3:55PM ERR ../go/src/app/internal/web/mid/errors.go:31 > ERROR occurred error="ent: constraint failed: constraint failed: UNIQUE constraint failed: users.email (2067)" req_id=2b2443a73146/Ka5V5SeNcl-000165
```

**Comment by dharvey242** [comment]
Just to note - i've got it working on both windows and ubuntu.  Just Debian where it fails for me.

**Reply by tankerkiller125** [reply]
Just as a confirmation (as we try to replicate this issue), is this on an ARM based processor? Or x86_64?

**Reply by katosdev** [reply]
Apologies for the issues that you have seen, I will be spending time to spin up my Raspberry Pi and conduct some testing. By any chance, can you confirm your Debian version, your Pi model, and as much detail about your config as possible so that I can try to replicate this one please?

**Reply by dharvey242** [reply]
Thanks. BCM2837B0 Arm A53. Buster version of Debian. Also tried it on the same processor but latest rpi image and the same thing.
